usecases/collect_nodes_statuses.py

```diff
@@ -0,0 +1,23 @@
+from typing import Union
+
+from common import config
+from domain.network.network import Network
+from gateways.aws.lambda_gateway import LambdaGateway
+from gateways.hathor.hathor_core_client import HathorCoreAsyncClient
+from gateways.hathor.hathor_core_gateway import HathorCoreGateway
+
+
+class CollectNodesStatuses:
+
+    def __init__(self, lambda_gateway: Union[LambdaGateway, None] = None) -> None:
+        self.lambda_gateway = lambda_gateway or LambdaGateway()
+
+    async def collect(self) -> None:
+        for node in config.hathor_nodes:
+            client = HathorCoreAsyncClient(node)
+            hathor_core_gateway = HathorCoreGateway(async_client=client)
+            await hathor_core_gateway.get_async('/v1a/status', self._send)
```

Comment:
We are making the for loop wait for each request to come back before proceeding to the next iteration, right?

Why don't we send them in parallel?

Reply:
Those `await` statements are a bit confuse, indeed. But it is happening asynchronously.